get_era5.py

```diff
@@ -0,0 +1,157 @@
+import cdsapi
+import yaml
+import pandas as pd
+from pathlib import Path
+
+c = cdsapi.Client()
+
+# Read case configuration
+with open("cases/era5_2021.yaml") as f:
+    config = yaml.safe_load(f)
+    
+# Use the preprocessing dates to select which dates to download
+datelist = pd.date_range(
+    start=config["preprocess_start_date"], end=config["preprocess_end_date"], freq="d"
+)
+
+#Construct level list
+if((config['levels'] == 'All') & (config['level_type'] == 'model_levels')):
+    levlist = '1/to/137'
+elif((config['levels'] == 'All') & (config['level_type'] == 'pressure_levels')):
+    levlist = '1/2/3/5/7/10/20/30/50/70/100/125/150/175/200/225/250/300/350/400/450/500/550/600/650/700/750/775/800/825/850/875/900/925/950/975/1000'
+else:
+    s = '/'
+    levlist = s.join(str(e) for e in config['levels'])
+
+# Construct list of downloading times
+if (config['data_freq'] == 'hourly'):
+    timlist  = '00:00:00/01:00:00/02:00:00/03:00:00/04:00:00/05:00:00/06:00:00/07:00:00/08:00:00/09:00:00/10:00:00/11:00:00/12:00:00/13:00:00/14:00:00/15:00:00/16:00:00/17:00:00/18:00:00/19:00:00/20:00:00/21:00:00/22:00:00/23:00:00'
+elif(config['data_freq'] == '3hourly'): 
+    timlist = '00:00:00/03:00:00/06:00:00/09:00:00/12:00:00/15:00:00/18:00:00/21:00:00'
+elif(config['data_freq'] == '6hourly'): 
+    timlist = '00:00:00/06:00:00/12:00:00/18:00:00'
+    
+area = config['data_area'] # North, West, South, East. Default: global
+grid = [config['data_resolution'], config['data_resolution']]  
+
+######################################################################
+# For model levels the following variables are downloaded:
+# u,v, q on selected levels
+# tp (total precipitation), e (evaporation) and sp (surface pressure)
+######################################################################
+if(config['level_type'] == 'model_levels'):
+    var_names = ["u","v","q"]
+    var_ids = [131,132,133]
+    
+    sfc_short = ['tp','e','sp']
+    sfc_long = ['mean_sea_level_pressure', 'evaporation', 'total_precipitation']
+    
+    for date in datelist:
+        dtstr = date.strftime("%Y-%m-%d")
+
+        #For first day of month, datadir should be created
+        data_dir = Path(config["input_folder"] + "/" + str(date.year) ).expanduser()
+        data_dir.mkdir(exist_ok=True, parents=True)
+        
+        #Subdirectory for every month 
+        data_dir = Path(config["input_folder"] + "/" + str(date.year) + '/' + str(date.month) ).expanduser()
+        data_dir.mkdir(exist_ok=True, parents=True)
+        
```

Comment:
Pathlib is very cool. Instead of making a path string with slashes (which might not work on windows), you can do:

`Path(input_folder).expanduser() / str(date.year) ` (note that the `/` is a division operator slash, not a string. 

So, you could reduce this to:

```suggestion
        # Create yearly parent dir if it doesn't exist yet
        parent = Path(config["input_folder"]).expand_user() / str(date.year)
        parent.mkdir(exist_ok=True, parents=True)
        
        # Create monthly subdirectories
        child = parent / str(date.month)
        child.mkdir(exist_ok=True)    # note: expanduser not necessary, already done above
```